packages/web-components/src/components/masthead/left-nav-overlay.ts

```diff
@@ -31,9 +31,12 @@ class DDSLeftNavOverlay extends LitElement {
     if (changedProperties.has('active')) {
       const doc = this.getRootNode() as Document;
       if (this.active) {
-        const masthead: HTMLElement | null = doc?.querySelector('dds-masthead');
+        // TODO: remove when masthead can account for banners above
+        const masthead: HTMLElement | null = doc?.querySelector('dds-cloud-masthead-container');
```

Comment:
I'm assuming that we'll have a follow-up PR that accounts for the offset top position of the `dds-masthead-container` if it exists. Just wanted to bring this up in case AEM implemented a banner treatment above the masthead (eg. Think banner, locale, etc).

Reply:
yeah that should be addressed in this feature request: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/7216